cli/remote.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+)
+
+func init() {
+	register("remote", runRemote, `
+usage: flynn remote <name> [-r <remote>] [-y]
```

Comment:
1. I think this should be `flynn remote add`
2. So far we always specify the app name with `flynn -a <app> remote add`, which doesn't exactly roll off the tongue and has positional issues, but we should probably support it.
3. It seems like the primary argument of this command should be the name of the remote, without having to specify `-r`.

Thoughts?